airflow-cluster/Jenkinsfile.template

```diff
@@ -1,13 +1,11 @@
-def final projectId = '@project_id@'
-def final componentId = 'airflow-worker'
+// See https://www.opendevstack.org/ods-documentation/ for usage and customization.
 
 @Library('ods-jenkins-shared-library@@ods_git_ref@') _
 
-// See https://www.opendevstack.org/ods-documentation/ods-jenkins-shared-library/latest/index.html for usage and customization.
 odsComponentPipeline(
     imageStreamTag: 'cd/jenkins-slave-airflow:@ods_image_tag@',
-    projectId: projectId,
-    componentId: componentId,
+    projectId: '@project_id@',
+    componentId: 'airflow-worker',
```

Comment:
why hardcoded?....

Reply:
No idea, it was already like this. @gerardcl @hugowschneider do you know?

Reply:
It is hardcoded for avoid the deployment of 2 airflow clusters in the same namespace. It is the only way (I found) for assuring that I can find the cluster and ensure a second one does not get deployed.

Another option would be through the provisioning app, but at the time this was to complex to be done.

Reply:
ok check then I m cool!